examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb

```diff
@@ -147,7 +147,7 @@
     "```scala\n",
     "import org.apache.spark.sql.SparkSession\n",
     "val spark = SparkSession.builder().appName(\"Taxi-GPU\").getOrCreate\n",
-    "%AddJar file:/data/libs/cudf-0.14-SNAPSHOT-cuda10.jar\n",
+    "%AddJar file:/data/libs/cudf-0.15-SNAPSHOT-cuda10.jar\n",
```

Comment:
Should we update the versions here?

Reply:
Updated the rest versions

Reply:
Why we need the SNAPSHOT string?

Reply:
It seems to be a "tech debt", they are removed now.